DOCSP-40959 Add Query Converter Test Bench Stub Page (#146)
mongodb/docs-relational-migrator: source/code-generation/query-converter-test-bench.txt

> Query Converter Test Bench

Reviewer: The UI doesn't use this name; it just says "Test queries". I think it would be better to avoid the internal Test Bench name in the docs.

Author: I dont think the UI has any labels or titles at all TBH, I see `Test queries` to the left of the UI, but that dosent look like an official title (like the Query Converter pane for example). Would it be possible to add a UI element for the name of the feature? This would make documenting and referring to it easier. 

A few examples:
- Query Converter Query Tester
- Query Converter Results Tester
- Query Converter Tester
- Query Results Comparison

....

<img width="1623" alt="Screenshot 2024-07-29 at 2 33 21 PM" src="https://github.com/user-attachments/assets/40c0f6ea-0d1d-4bba-a9c9-311eb47ab03c">

Reviewer: As discussed in the call - the labelling of the feature in the product was a conscious design decision. Given the pane is labelled "Test queries" I'd prefer to use the same name in the docs. If we need to nounify it we can call it the "Test queries pane".

==========================

.. meta::
   :keywords: relational migrator, query converter test bench

.. facet::
   :name: genre
   :values: reference

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

Use Query Converter Test Bench to test SQL queries 
and stored procedures converted by :ref:`rm-query-converter`. 
Query Converter Test Bench uses the project's source 

> and destination databases connections and allows you to

Reviewer 2: ```suggestion
and destination database connections and allows you to
```

run the original SQL and converted MongoDB code.
The results from each query are displayed on the Query Converter 

> Test Bench allowing for a side-by-side comparison.

Reviewer 2: ```suggestion
Test Bench, allowing for a side-by-side comparison.
```

Use Cases
---------

> Query Converted Test Bench allows you to:

Reviewer: Typo

Author: Fixed.

   :depth: 1

>    :class: singlecol

Reviewer 2: [Taxonomy]

Sorry I missed this earlier but could we add the taxonomy tags here too?

Reviewer 2: Thank you!

Use Query Converter Test Bench to test SQL queries 
and stored procedures converted by :ref:`rm-query-converter`. 

> Query converter test bench uses the connections defined for your 

Reviewer 2: [Capitalization]
This applies throughout since it is the product name.

```suggestion
Query Converter Test Bench uses the connections defined for your 
```

source and destination databases and allows you to run 

> the original and converted queries or stored procedures.

Reviewer 2: [Suggestion]
This sentence feels a little complex may we could try something like: "Query Converter Test Bench connects to source and destination databases that allow you to ..."

> The results from each display on the Query Converter 

Reviewer 2: ```suggestion
The results from each query are displayed on the Query Converter 
```

Test Bench allowing for a side-by-side comparison.

Use Cases
---------

> Query converted test bench allows you to:

Reviewer 2: ```suggestion
Query Converted Test Bench allows you to:
```

- Compare your original and converted 

>   query results conveniently in a single place.

Reviewer 2: [minimalism]

```suggestion
  query results in a single place.
```

- Increase your confidence in code automatically 
  converted by the query converter.
- See potential logic errors and edit the converted 
  code before moving it to production.

Requirements
------------

To use Query Converter Test Bench, you must 
run the Query Runner Docker container. This 
container can be run locally or hosted remotely. 
For details, see :ref:`rm-install-query-runner`.

Get Started
-----------

- :ref:`rm-install-query-runner` 
- :ref:`rm-compare-converted-queries`

> 

Reviewer 2: [Suggestion]
Could we add the Learn More section and link to the Query Converter page here again?

Reviewer 2: Makes senses, thanks!
